easybuild/easyconfigs/g/git-lfs/git-lfs-2.7.1.eb

```diff
@@ -0,0 +1,35 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+
+easyblock = "Tarball"
+
+name = 'git-lfs'
+version = '2.7.1'
+
+homepage = 'https://git-lfs.github.com/'
+description = """Git Large File Storage (LFS) replaces large files such as audio samples, videos, 
+ datasets, and graphics with text pointers inside Git, while storing the file contents on a remote server 
+ like GitHub.com or GitHub Enterprise."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['https://github.com/git-lfs/git-lfs/releases/download/v%(version)s/']
+sources = ['%(name)s-linux-amd64-v%(version)s.tar.gz']
```

Comment:
@vanzod Can we build this from source instead?

Maybe it's also worth having a simple sanity check command (certainly when installing it as a binary)...